Cargo.toml

```diff
@@ -29,7 +29,7 @@ memmap = "0.6"
 num = "0.1"
 num_cpus = "1.8"
 petgraph = "0.4"
-rand = "0.4"
+rand = "0.5"
```

Comment:
To get this to run I had to do rand="0.5.0-pre.1"  Do not change this, we will wait until the release to merge the PR.